.github/workflows/docker_update_toolbox.yml

```diff
@@ -13,23 +13,42 @@ on:
     paths:
       - 'images/toolbox/**'
 
+env:
+  REGISTRY_IMAGE: "antrea/toolbox"
+
 jobs:
-  build:
+  get-docker-tag:
     runs-on: ubuntu-latest
     steps:
     - name: Check-out code
       uses: actions/checkout@v4
       with:
         fetch-depth: 0
-    - name: Set docker tag
+    - name: Get docker tag
+      id: get-outputs
       working-directory: images/toolbox
       run: |
         version=$(head -n 1 VERSION)
         version_update_sha=$(git log -n 1 --pretty=format:%H -- VERSION)
         rev=$(git rev-list --count ${version_update_sha}..HEAD -- .)
-        echo "version=${version}" >> $GITHUB_ENV
-        echo "rev=${rev}" >> $GITHUB_ENV
-        echo "docker_tag=${version}-${rev}" >> $GITHUB_ENV
+        echo "version=${version}" >> $GITHUB_OUTPUT
+        echo "rev=${rev}" >> $GITHUB_OUTPUT
+        echo "docker_tag=${version}-${rev}" >> $GITHUB_OUTPUT
+    outputs:
+      version: ${{ steps.get-outputs.outputs.version }}
+      rev: ${{ steps.get-outputs.outputs.rev }}
```

Comment:
this seems not used

Reply:
yes, it's true but I didn't feel the need to remove it